Assets/Scripts/Threats/FixedWingThreat.cs

```diff
@@ -3,6 +3,13 @@
 using UnityEngine;
 
 public class FixedWingThreat : Threat {
+  [SerializeField]
+  private float _navigationGain = 50f;
```

Comment:
is it worth putting this in static config?

Reply:
Absolutely yes. Meant to write a TODO there